examples/xgboost_cv.py

```diff
@@ -0,0 +1,99 @@
+"""
+Optuna example that performs Cross Validation for cancer dataset using XGBoost.
+
+In this example, we perform cross-validation to accuracy of cancer detection
+using XGBoost. We optimize both the choice of booster model and their hyper
+parameters.
+
+We have following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python xgboost_cv.py
+
+
+"""
+
+
+import os
+import shutil
+import sklearn.datasets
+import sklearn.metrics
+import xgboost as xgb
+import optuna
```

Comment:
Please add a blank line before optuna and the third party libraries to pass the `black` checker. In addition, please check https://github.com/optuna/optuna/pull/1202#discussion_r464736130 on `examples/xgboost_simple.py`.
```suggestion
import xgboost as xgb

import optuna
```

Reply:
made the changes as you suggested but still get ci failing.

Reply:
Hmm, I cannot see the blank line in your commit. My suggestion is out-dated, so could you add a blank line manually?